README.md

```diff
@@ -1,24 +1,405 @@
+- Tyler Blackmon - [GitHub](https://github.com/tblackmon-tiel) || [LinkedIn](www.linkedin.com/in/tyler-blackmon/)
 
-* Configuration
+**Frontend**
 
-* Database creation
+- Devin Altobello - [GitHub](https://github.com/daltobello) || [LinkedIn](https://www.linkedin.com/in/devin-altobello-2100036b/)
+- Jen Ziel - [GitHub](https://github.com/jenziel) || [LinkedIn](https://www.linkedin.com/in/jen-ziel400/)
+- Marisa Wyatt - [GitHub](https://github.com/Marisa5280) || [LinkedIn](https://www.linkedin.com/in/marisarwyatt/)
 
-* Database initialization
+## Technologies
+Primary:
+- Ruby 3.2.2
+- Rails 7.0.8
+- PostgreSQL
 
-* How to run the test suite
+Production Gems:
+- [jsonapi-serializer](https://github.com/jsonapi-serializer/jsonapi-serializer)
+- [Faraday](https://lostisland.github.io/faraday/#/)
 
-* Services (job queues, cache servers, search engines, etc.)
+Testing Gems:
+- [rspec-rails](https://github.com/rspec/rspec-rails)
+- [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers)
+- [FactoryBot](https://github.com/thoughtbot/factory_bot)
+- [Faker](https://github.com/faker-ruby/faker)
 
-* Deployment instructions
+## Installation/Setup
+### Cloning and installing dependencies
+- `git clone <repo_name>`
+- `cd <repo_name>`
+- `bundle install`
+- `rails db:{drop,create,migrate}`
 
-* ...
+### Third Party APIs
+Experience Exchange uses two third party APIs - [Geoapify](https://apidocs.geoapify.com/docs/geocoding/forward-geocoding/#about) for geocoding, and [Unsplash](https://unsplash.com/documentation) for randomized user images. Only Geoapify needs to be setup in the primary repo for geocoding of a user's address.
+
+Follow the Geoapify documentation to get an API key, and place it in the rails credentials file as follows:
+- `EDITOR="code --wait" rails credentials:edit`
+```
+geoapify:
+  api_key: <YOUR_CLIENT_ID>
+```
+
+### Setting up data
+The seed file already contains a small amount of test data that can be initialized with `rails db:seed`. Alternative data can be added via editing the seed file, or manual creation in the rails console (`rails c`).
+
+- Users: first_name, last_name, email (unique), street, city, state, zip, is_remote, about (optional: lat, lon)<br>
+`User.create(first_name: "Steve", last_name: "Jobs", email: "steve@gmail.com", street: "1234 Street", city: "Cupertino", state: "CA", zipcode: "12345", lat: "1.12", lon: "1.12", is_remote: "true", about: "I am a very good programmer")`
+
+- Meetings: date, start_time, end_time, purpose<br>
+`Meeting.create!(date: "2023-12-15", start_time: "07:30", end_time: "08:00", purpose: "testing")`
+
+- Skills: name, proficiency, user_id<br>
+`Skill.create!(name: "Testing", proficiency: 5, user_id: 1)`
+
+- User_Meetings: user_id, meeting_id, is_requestor<br>
+`UserMeeting.create!(user_id: 1, meeting_id: 1, is_requestor: true`
+
+## Testing
+There are two options for testing, either the existing test suite, or manual testing with Postman.
```

Comment:
Do we need to add any commentary on the dependency for the image microservice? I guess if it's not running they will just get nil image.